deployments/tyk2/docker-compose.yml

```diff
@@ -1,15 +1,15 @@
 version: '3.3'
 services:
   tyk2-dashboard:
-    image: tykio/tyk-dashboard:latest
+    image: tykio/tyk-dashboard:${DASHBOARD_VERSION:-latest}
```

Comment:
What do you think about using env vars for the images on the other `tyk2` components? Like the Gateway and Pump

Reply:
there are lots. didnt want to add more before you review this